src/lib/components/table/components/TableContents/Collapsible.tsx

```diff
@@ -1,30 +1,31 @@
-import { ReactNode } from "react";
-import AnimateHeight from "react-animate-height";
-import { useMediaQuery } from "../../../../hooks/useMediaQuery";
-
+import { ReactNode, useEffect, useRef, useState } from 'react';
+import styles from './Collapsible.module.scss';
 interface CollapsibleProps {
   children: ReactNode;
   isExpanded?: boolean;
-  isMobile?: boolean;
 }
-  
+
 export const Collapsible = ({ children, isExpanded }: CollapsibleProps) => {
-  const isDesktop = useMediaQuery('ABOVE_DESKTOP');
+  const [height, setHeight] = useState<number | undefined>();
 
-  if (!isDesktop) {
-    return (
-      <div>
-        {isExpanded ? children : null}
-      </div>
-    );
-  }
+  const containerRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    if (containerRef.current && height === undefined) {
+      const scrollheight = containerRef.current.scrollHeight;
+      setHeight(scrollheight);
```

Comment:
This needs to be recalculated on screen change as it affects the maxHeight and if starting on tablet/desktop and resizing the page, it hides some of the  content